configure.ac

```diff
@@ -16,13 +16,12 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 AC_INIT([Cockpit],
-        [m4_esyscmd([tools/git-version-gen .tarball])],
+        [m4_esyscmd_s([echo $(git describe | tr - . )])],
         [devel@lists.cockpit-project.org],
         [cockpit],
         [https://cockpit-project.org/])
 
-# The above git-version-gen command can either use .git or a .tarball file
```

Comment:
I am not quiet sure, how it would work with that `.tarball` since it still did `git describe`. So I dropped that, but I can surely put there `git -C $(dirname .tarball) describe`.

Reply:
Oh, good point, I forgot about the .tarball bit, sorry! Dropping that would mean that you couldn't ./autogen.sh on a released tarball. So we do need an alternative `|| cat .tarball`, but that's a bit tricky to work into this command line. You'd need `set -o pipefail` or the `if`, or maybe just leave git-version-gen as a separate file if it becomes too clumsy.

Reply:
Right, so I changed it to check if is in git repository. If not then try to `cat .tarball`. That is how it worked before.